compiler/ballerina-lang/src/main/resources/compiler.properties

```
@@ -1948,3 +1948,7 @@ error.int.range.overflow=\
 
 error.unsupported.usage.of.default.values.for.key.field.in.table.member=\
   using default values of record fields for table key fields is not yet supported
+
+error.cannot.use.wildcard.binding.pattern.for.error.cause=\
+  cannot use a wildcard binding pattern to bind the cause: a wildcard binding pattern can be used only with a value \
+  that belong to type ''any''
```

```suggestion
  that belongs to type ''any''
```

Sorry, missed this earlier.

---

8d79ea8